emulator/index.js

```diff
@@ -0,0 +1,32 @@
+// Require Mock Mobile
+require('../thali/mockmobile');
+var ThaliReplicationManager = require('../thali/thalireplicationmanager');
+var express = require('express');
+var expressPouchDB = require('express-pouchdb');
+var PouchDB = require('pouchdb');
+var path = require('path');
+var os = require('os');
+
+// Set up PouchDB defaults for leveldown
+PouchDB.defaults({db: require('leveldown'), prefix: dbPath});
```

Comment:
I'm confused. Isn't leveldown the default adapter for PouchDB? We shouldn't have to specify it with require should we?